packages/@internationalized/date/tests/manipulation.test.js

```diff
@@ -37,7 +37,7 @@ describe('CalendarDate manipulation', function () {
 
     it('should add months and constrain days', function () {
       let date = new CalendarDate(2020, 8, 31);
-      expect(date.add({months: 1})).toEqual(new CalendarDate(2020, 9, 30));
+      expect(date.add({months: 1})).toEqual(new CalendarDate(2020, 10, 1));
```

Comment:
It should only add 1 month, regardless of number of days, so I think Sept 30 was correct.